src/Elastic.Apm/Helpers/StacktraceHelper.cs

```diff
@@ -48,7 +48,7 @@ internal static class StacktraceHelper
 			var len = stackTraceLimit == -1 ? frames.Length : Math.Min(frames.Length, stackTraceLimit);
 			var retVal = new List<CapturedStackFrame>(len);
 
-			logger.Trace()?.Log("transform stack frames");
+			logger?.Trace()?.Log("transform stack frames");
```

Comment:
The null conditional operator on `logger` looks unneccessary. `StacktraceHelper` is internal and `GenerateApmStackTrace` is called only by agent code where the logger should never be null (if it is null, it's a bug).

Reply:
Ok, there were some calls on `logger` with `.?` and some without it - I removed it from all of them.